modules/manage/pages/manage-logging/manage-logging.adoc

```diff
@@ -444,3 +446,54 @@ The Logging REST API provides the endpoints for retrieving log and diagnostic in
 
 To retrieve log information use the `/diag` and `/sasl_logs`
 xref:rest-api:logs-rest-api.adoc[REST endpoints].
+
+[#adjust-threshold-slow-op-logging]
+== Adjusting Threshold for Logging Slow Operations
+
+It is possible to examine and/or alter the logging threshold for slow-running operations.
+This is done using the `mcctl` command that comes packaged with the Couchbase server installation.
+
+=== Getting Threshold Details
+The current settings are retrieved by using the `mcctl` cli to execute the `get sla` command:
+
+.Getting threshold details
+[source, bash]
+----
+/opt/couchbase/bin/mcctl --host localhost -u Administrator -P password \
+get sla
+----
+
+.Result
+[source, json]
+----
+{"comment":"Current MCBP SLA configuration",
+"version":1,
+"default":{"slow":"500 ms"}},
+"COMPACT_DB":{"slow":"1800 s"},
+"DELETE_BUCKET":{"slow":"10 s"},
+"SEQNO_PERSISTENCE":{"slow":"30 s"}
+}
+----
+
+The JSON message returned gives details of the operation being logged and the threshold time that will cause a timing message to be logged.
+
+=== Setting the Threshold
+
+The `mcctl` command line interface is also used to set the thresholds for the memcahe operations:
+
+.Set logging threshold example
+[source, bash]
+----
+/opt/couchbase/bin/mcctl --host localhost -u Administrator -P password \
+set sla '{"version":1, "DELETE_BUCKET":{"slow":"100ms"}}'
```

Comment:
Above there are spaces between the number and the strings, but here there is not a space.  Typo?

Reply:
Not a type. The `get sla` command formats the strings to add in the space.

Reply:
Up to you if you want to mention this.  Maybe it's not worth it.  The expectation is that you would read-what-you-write and vice versa.  Since this interface doesn't follow that principle, it may or may not be worth mentioning.  Obviously not a fault of the documentation, but I bet someone will get it wrong.  Hopefully it just fails in a reasonable way.

Reply:
I'm going to quickly try doing a set sla with the space. It might worth both ways.

Reply:
Hah! Of course it works both ways. The comments in the code didn't indicate it, but it does makes sense to parse with either a space or no space. Right,  I'm going to adjust the text to put spaces in the set examples, then add a tip to the users know that it will also work without spaces as a small convenience.

No need for another review. 

I'm going to put the change in version 7.0 too, since 7.1 has only just been released.